docs/concepts/storage/dynamic-provisioning.md

```diff
@@ -0,0 +1,122 @@
+---
+approvers:
+- saad-ali
+title: Dynamic Volume Provisioning
+---
+
+{% capture overview %}
+
+Dynamic volume provisioning allows storage volumes to be created on-demand.
+Without dynamic provisioning, cluster administrators have to manually make
+calls to their cloud or storage provider to create new storage volumes, and
+then create [`PersistentVolume` objects](/docs/concepts/storage/persistent-volumes/)
+to represent them in Kubernetes. The dynamic provisioning feature eliminates
+the need for cluster administrators to pre-provision storage. Instead, it
+automatically provisions storage when it is requested by users.
+
+{% endcapture %}
+
+{:toc}
+
+{% capture body %}
+
+## Background
+
+The implementation of dynamic volume provisioning is based on the API object `StorageClass`
+from the API group `storage.k8s.io`. A cluster administrator can define as many
+`StorageClass` objects as needed, each specifying a *volume plugin* (aka
+*provisioner*) that provisions a volume and the set of parameters to pass to
+that provisioner when provisioning.
+A cluster administrator can define and expose multiple flavors of storage (from
+the same or different storage systems) within a cluster, each with a custom set
+of parameters. This design also ensures that end users don’t have to worry
+about the the complexity and nuances of how storage is provisioned, but still
+have the ability to select from multiple storage options.
+
+More information on storage classes can be found
+[here](/docs/concepts/storage/persistent-volumes/#storageclasses).
```

Comment:
This looks like a dead link?

Reply:
no. it translates to https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storageclasses